jetbrains-rider/ReSharper.AWS/src/AWS.Psi/Lambda/LambdaPsiHost.cs

```diff
@@ -39,8 +40,18 @@ public class LambdaPsiHost
             _locks = locks;
             _logger = logger;
 
+            if (solution == null)
+            {
+                throw new Exception("AAA - solution is null");
```

Comment:
AAA? Did you mean to add better messages here?

Reply:
Hmm... Debugging code :) deleted.